rca/events/models.py

```diff
@@ -763,8 +764,8 @@ def get_ics_record(self):
             # Get date
             date = self.start_date + datetime.timedelta(days=day)
             # Get times
-            start_datetime = datetime.datetime.combine(date, self.start_date.time())
-            end_datetime = datetime.datetime.combine(date, self.end_date.time())
+            start_datetime = datetime.datetime.combine(date, self.start_time)
```

Comment:
This throws a `TypeError` (combine() argument 2 must be datetime.time, not None) when the `start_time` field is empty (it is not a mandatory field).

Reply:
Resolved in https://github.com/torchbox/rca-wagtail-2019/commit/26b2ee410ef2d5ef2d147cea795d4f44bdbc733d